src/components/InstancesList/InstancesList.js

```diff
@@ -365,6 +367,15 @@ class InstancesView extends React.Component {
     } = this.props;
 
     const resultsFormatter = {
+      'select': () => (
+        <FormattedMessage id="ui-inventory.instances.rows.select">
+          {ariaLabel => (
+            <CheckboxColumn>
+              <Checkbox aria-label={ariaLabel} />
+            </CheckboxColumn>
+          )}
+        </FormattedMessage>
```

Comment:
`aria-label` needs to be a string but `<FormattedMessage>` with render-props provides an array in v5. Pull `intl { formatMessage }` out of `this.props` above then use 
```
<Checkbox aria-label={formatMessage({ id: 'ui-inventory.instances.rows.select' })} />

```

Reply:
As I see in the browser the string is applied correctly even with the form I used. I think that in this case the array is converted to a string though.
```
<input id="checkbox-40" type="checkbox" aria-label="Select instance" class="input---2C-_P" aria-invalid="false" value="">
```
I saw the similar PR you made with the changes you suggested. Do you think this render prop should not be used at all?

Reply:
Per discussion with Zak, it is not recommended because of the prop-types warnings where `ariaLabel` is array, but `aria-label` expects string.